app/importers/market_research_data.rb

```diff
@@ -3,9 +3,9 @@
 
 class MarketResearchData
   include Importable
-  include VersionableResource
```

Comment:
Ok, good point. In fact, I need to think about how VersionableResource is going to work with Industry Mapper in the mix... feel free to ignore my comment.